src/providers/data_provider_fo.c

```diff
@@ -675,6 +683,12 @@ errno_t be_resolve_server_process(struct tevent_req *subreq,
         DEBUG(SSSDBG_FUNC_DATA, "Found address for server %s: [%s] TTL %d\n",
               fo_get_server_str_name(state->srv), ipaddr,
               srvaddr->addr_list[0]->ttl);
+    } else {
+        DEBUG(SSSDBG_CRIT_FAILURE, "Missing server name.\n");
+        dump_be_svc_data(state->svc);
+        dump_fo_server(state->srv);
```

Comment:
Won't `dump_fo_server_list()` also log the server `state->srv`? Why also logging it separately here?

Reply:
Hi,

with just dumping the list it would not be clear which list entry corresponds to `state->srv`. Additionally, since something unexpected it going on, `state->srv` might still have back-links to list entries but is not on the list anymore, i.e. no other list items will have links to `state->srv`.

bye,
Sumit